src/main/java/post_reply_user/common_User.java

```diff
@@ -0,0 +1,86 @@
+package post_reply_user;
+
+import java.util.ArrayList;
+class common_User extends User{
+
+    ArrayList<post_reply_user.Post> allPost;
+    ArrayList<Reply> allReply;
+
+    public common_User(String nick_name,String pass_word, String avatar){
+        super(nick_name, pass_word, avatar);
+        setallPost();
+        setallReply();
+
+    }
+
+    public void setallPost() {
+        this.allPost = new ArrayList<>();
+    }
+
+    public void setallReply() {
+        this.allReply = new ArrayList<>();
+    }
+
+    public String get_nickname(User user_3){
+        return user_3.nickname;
+    }
+
+    public String get_password(User user_3){
+        return user_3.password;
+    }
+
+    public String get_avatar(User user_3){
```

Comment:
Since you have get_avatar method, you should add set_avatar method to allow user to change